ckanext/canada/tests/test_validators.py

```diff
@@ -24,19 +29,17 @@ def test_simple(self):
     def test_too_short(self):
         with pytest.raises(Invalid) as ie:
             canada_tags(u'h', {})
-        err = ie.value
-        expected = {}
-        #TODO: assert the expected error
-        assert ie is not None
+        err = str(ie.value)
+        expected = 'length is less than minimum'
+        assert expected in err
```

Comment:
```suggestion
        assert 'length is less than minimum' in err
```
writing tests this way makes it easier to see exactly what failed directly in the traceback

Reply:
all singles string `expected` variables have been removed, and the string has been moved directly into the `assert` statements.